flink-scala/src/main/scala/org/apache/flink/api/scala/package.scala

```diff
@@ -71,12 +71,11 @@ package object scala {
           "supported on Case Classes (for now).")
     }
   }
+
   def getCallLocationName(depth: Int = 3) : String = {
     val st = Thread.currentThread().getStackTrace()
-    if(st.length < depth) {
-      return "<unknown>"
-    }
-    st(depth).toString
+    if (st.length < depth) "<unknown>"
```

Comment:
Our [code style](http://flink.apache.org/contribute-code.html#code-style) requires curly braces for `if` statements.